Library/Homebrew/test/livecheck/strategy/sparkle_spec.rb

```diff
@@ -189,20 +349,20 @@
         sparkle.versions_from_content(xml[:appcast]) do |item|
           item.nice_version&.sub("1", "0")
         end,
-      ).to eq([subbed_items[0]])
+      ).to eq([subbed_versions_in_sort_order[0]])
```

Comment:
```suggestion
      ).to eq([subbed_items[0]])
```

Reverting in relation to the changes above.